rasa/nlu/featurizers/count_vectors_featurizer.py

```diff
@@ -158,28 +185,37 @@ def __init__(
         self._check_analyzer()
 
         # declare class instance for CountVectorizer
-        self.vectorizer = vectorizer
-
-    def _get_message_text(self, message):
-        if message.get("spacy_doc"):  # if lemmatize is possible
-            text_tokens = [t.lemma_ for t in message.get("spacy_doc")]
-            # text = " ".join([t.lemma_ for t in message.get("spacy_doc")])
-        elif message.get("tokens"):  # if directly tokens is provided
-            text_tokens = [t.text for t in message.get("tokens")]
+        if vectorizer:
+            self.vectorizer = vectorizer
+
+    def _get_message_text_by_attribute(self, message, attribute=MESSAGE_TEXT_ATTRIBUTE):
```

Comment:
this def needs to be split - hard to read and follow esp. due to the added indirections (`MESSAGE_SPACY_FEATURES_NAMES`)